docs/build-wallets.md

```diff
@@ -39,12 +37,12 @@ application.
 | [Ownbit](https://ownbit.io/)                                       | Live              | Ownbit       | iOS and Android                        |
 | [AirGap](https://airgap.it/)                                       | Live              | AirGap       | IOS, Android, MacOS, Windows and Linux |
 | [Cobo Wallet](https://cobo.com/)                                   | Building          | Cobo         | IOS and Android                        |
-| [SafePal](https://www.safepal.io/)                                 | Live              | SafePal      | Hardware                               |
+| [SafePal](https://www.safepal.io/download)                         | Live              | SafePal      | Hardware                               |
 | [Polkadot{.js}](https://github.com/EthWorks/extension)             | Building          | EthWorks     | Browser extension                      |
 | [MetaMask](https://metamask.io/index.html)                         | Building          | MetaMask     | IOS, Android, and browser extension    |
 | [Speckle](https://github.com/GetSpeckle/speckle-browser-extension) | Building          | Speckle      | Browser extension                      |
-| [Enzyme](https://getenzyme.dev/)                                   | Building          | BlockXLabs   | Browser extension                      |
 | [Ledger App](https://zondax.ch/kusama.html#overview)               | Building          | Zondax       | Hardware                               |
 | [KodaDot](https://kodadot.netlify.app/#/accounts)                  | Building          | Geefu        | Browser                                |
 | [Subwallet](https://github.com/yxf/subwallet)                      | Building          | yxf          | CLI Wallet                             |
 | [Fearless](https://soramitsu.co.jp/fearless)                       | Building          | Soramitsu    | IOS and Android                        |
+| [Crypto.com App](https://crypto.com/en/index.html)                 | Live              | Crypto.com   | IOS and Android                        |
```

Comment:
Just a nitpick, but could the all the Live wallets be clustered together and the building ones clustered? I think it makes the table more organized.

Reply:
Good idea!

Reply:
done